src/VirtualAlexa.ts

```diff
@@ -217,9 +229,9 @@ export class VirtualAlexaBuilder {
 
         let interactor;
```

Comment:
I would add the logic to select the default locale here if it is not set. Then it can be a required parameter elsewhere. As it is not something that is truly optional, better to default it earlier in the code paths.